MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -4395,6 +4399,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D65A6FEB1E40002800C69FBA /* msal__framework__mac.xcconfig */;
 			buildSettings = {
+				CODE_SIGN_IDENTITY = "-";
```

Comment:
Just curious, what does it mean by adding a dash here? :)

Reply:
This means Ad Hoc signing.:)